src/pages/EventPage/components/StatusLabel/StatusLabel.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Container, Label, HelpText } from './styled';
+import { EVENT_STATUS_OPTIONS } from 'src/utils/constants'
+
+type StatusLabelProps = {
+  eventStatus: string;
+};
+
+function StatusLabel({ eventStatus }: StatusLabelProps) {
+  if (eventStatus === EVENT_STATUS_OPTIONS.Draft)
+    return (
+      <Container>
+        <Label color="grey">Draft</Label>
+        <HelpText>This event is only visible to you</HelpText>
+      </Container>
+    );
+  else {
+    return (
+      <Container>
+        <Label color="green">Published</Label>
+      </Container>
+    );
+  }
```

Comment:
Here there's two way to write this in a more idiomatic way. Since you returns in both cases then it's possible to omit the else-branch like how it's done in the "early return comment". Alternatively, it is also typical to use ternaries inside of the JSX, e.g.:
```tsx
return (
  <Container>
    {eventStatus === EVENT_STATUS_OPTIONS.DRAFT ? (
      <Label color="grey">Draft</Label>
      <HelpText>This event is only visible to you</HelpText>
    ) : (
      <Label color="green">Published</Label>
    )}
  </Container>
);
```
Although it might not be as readble (heavily depends on the syntax highlighting of your editor).

Reply:
Which of the two approaches do you recommend using in this case and in general?

Reply:
I usually default to using the latter approach, but if it becomes difficult to read then I'll use the former - so it depends! 😝 But, after trying it out in my editor then I realized that I wasn't really satisfied with either approach, so I'd probably choose a third one that sacrifices a smal bit of performance for readability:
```jsx
  const isDraft = eventStatus === EVENT_STATUS_OPTIONS.Draft;
  const label = isDraft ? 'Draft' : 'Published';
  return (
    <Container>
      <Label color="grey">{label}</Label>
      {isDraft && <HelpText>This event is only visible to you</HelpText>}
    </Container>
  );
```